Makefile

```diff
@@ -148,11 +148,11 @@ $(PROGS): $(GO_FILES)
 bins: generate $(PROGS)
 
 .PHONY: run
-run: examples/example-gateway/build/example-gateway
+run: examples/example-gateway/build/services/example-gateway/example-gateway
```

Comment:
Do we want to put the binary in `build/services/{{service}}/{{service}}` or should we have a `build/bin/{{service}}` or `bin/{{service}}` ?